create_phone_number_test.py

```diff
@@ -0,0 +1,5 @@
+from create_phone_number import create_phone_number
+
+def test_create_phone_number():
+    assert  create_phone_number([1, 2, 3, 4, 5, 6, 7, 8, 9]) == '(123) 456-789'
+    assert  create_phone_number([1, 'b', 3, 4, 5, 'a', 7, 8, 9]) == '(1b3) 45a-789'
```

Comment:
I see what you've done here, you are correct your code works with letters to, but since the challenge was:

> Write a function that accepts an array of 10 integers (between 0 and 9), that returns a string of those numbers in the form of a phone number.

`array of 10 integers` I'd stick with that for the tests.